activesupport/lib/active_support/message_encryptor.rb

```diff
@@ -115,7 +115,7 @@ def _decrypt(encrypted_message)
         # Currently the OpenSSL bindings do not raise an error if auth_tag is
         # truncated, which would allow an attacker to easily forge it. See
         # https://github.com/ruby/openssl/issues/63
-        raise InvalidMessage if aead_mode? && auth_tag.bytes.length != 16
+        raise InvalidMessage if aead_mode? && (auth_tag.nil? || auth_tag.bytes.length != 16)
```

Comment:
This was something I stumbled upon during testing the upgrade cases. It is probably something that should be pulled out into a separate PR and backported to whenever AEAD was added as a feature of `MessageEncryptor`.